src/server/server.ts

```diff
@@ -196,8 +202,14 @@ export default class Server {
     this.api.on('connection', (spark: Primus.spark) => {
       console.info('API', 'CON', 'Open:', spark.address.ip)
 
-      spark.on('hello', (data: NodeData) => {
-        const {stats, proof}: { stats: Stats, proof: Proof } = data
+      spark.on('hello', (data: NodeResponse) => {
+        const {
+          stats,
+          proof
+        } = <{
+          stats: Stats,
```

Comment:
is this a WrappedResponse?

Reply:
no, it is only wrapped when stats contains stats, but this time stats contains the data itself directly.